packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+/// An iOS style button.
+///
+/// Takes in a text or an icon that fades out and in on touch. May optionally have a
+/// background.
+///
+/// See also:
+///
+///  * <https://developer.apple.com/ios/human-interface-guidelines/ui-controls/buttons/>
+class CupertinoButton extends StatefulWidget {
+  // TODO(xster): move this to a common Cupertino color palatte with the next yak.
+  static const Color kCupertinoBlue = const Color(0xFF007AFF);
+  static const Color kCupertinoWhite = const Color(0xFFFFFFFF);
+  static const Color kCupertinoDisabledBackground = const Color(0xFFA9A9A9);
+  static const Color kCupertinoDisabledForeground = const Color(0xFFC4C4C4);
+
+  static const TextStyle _kCupertinoButtonStyle = const TextStyle(
+    fontFamily: '.SF UI Text',
+    inherit: false,
+    fontSize: 15.0,
+    fontWeight: FontWeight.normal,
+    color: kCupertinoBlue,
+    textBaseline: TextBaseline.alphabetic,
+  );
+
+  static final TextStyle _kCupertinoDisabledButtonStyle = _kCupertinoButtonStyle.copyWith(
+    color: kCupertinoDisabledForeground,
+  );
+
+  static final TextStyle _kCupertinoBackgroundButtonStyle = _kCupertinoButtonStyle.copyWith(
+    color: kCupertinoWhite,
+  );
+
+  static const EdgeInsets _kCupertinoButtonPadding = const EdgeInsets.all(16.0);
+  static const EdgeInsets _kCupertinoBackgroundButtonPadding =
+      const EdgeInsets.symmetric(vertical: 16.0, horizontal: 64.0);
```

Comment:
If I understand right, doesn't having them at the file level make them global (library) fields once the file is imported? 

Don't really have a strong opinion but maybe once we have 20 composable cupertino files, it might be annoying for autocompleters to give _kButtonInnerPadding and _kPrimaryTextFontSize etc that you imported as soon as you type '_k' when you're in CupertinoScaffold and want something qualified like CupertinoPicker._kHeight